doc/source/release-notes/Percona-Server-5.6.24-72.2.rst

```diff
@@ -0,0 +1,42 @@
+
+Based on `MySQL 5.6.24 <http://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-24.html>`_, including all the bug fixes in it, |Percona Server| 5.6.24-72.2 is the current GA release in the |Percona Server| 5.6 series. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.6.24-72.2 milestone at Launchpad <https://launchpad.net/percona-server/+milestone/5.6.24-72.2>`_.
+
+New Features
+============
+
+ TokuDB storage engine package has been updated to version `7.5.7 <http://docs.tokutek.com/tokudb/tokudb-release-notes.html#tokudb-version-7-x>`_.
+ 
+Bugs Fixed
+==========
+
+ A server binary as distributed in binary tarballs could fail to load on different systems due to an unsatisfied ``libssl.so.6`` dynamic library dependency. This was fixed by replacing the single binary tarball with multiple tarballs depending on the *OpenSSL* library available in the distribution: 1) ssl100 - for all *Debian/Ubuntu* versions except *Squeeze/Lucid* (``libssl.so.1.0.0 => /usr/lib/x86_64-linux-gnu/libssl.so.1.0.0 (0x00007f2e389a5000)``); 2) ssl098 - only for *Debian Squeeze* and *Ubuntu Lucid* (``libssl.so.0.9.8 => /usr/lib/libssl.so.0.9.8 (0x00007f9b30db6000)``); 3) ssl101 - for *CentOS 6* and *CentOS 7* (``libssl.so.10 => /usr/lib64/libssl.so.10 (0x00007facbe8c4000)``); 4) ssl098e - to be used only for *CentOS 5* (``libssl.so.6 => /lib64/libssl.so.6 (0x00002aed5b64d000)``). Bug fixed :bug:`1172916`.
+ 
+ Execution of stored procedures with subqueries leads to memory leaks. Bug fixed :bug:`1380985` (upstream :mysqlbug:`76349`).
+
+ Multi-threaded replication on slave would break with an ``1755`` error after restarting the server a few times. This is a regression introduced by fix for bug :bug:`1331586` in :rn:`5.6.21-70.0`.
+
+ A string literal containing an invalid UTF-8 sequence could be treated as falsely equal to a UTF-8 column value with no invalid sequences. This could cause invalid query results. Bug fixed :bug:`1247218` by a fix ported from *MariaDB* (`MDEV-7649 <https://mariadb.atlassian.net/browse/MDEV-7649>`_).
+
+ |Percona Server| ``.deb`` binaries were built without fast mutexes. Bug fixed :bug:`1433980`.
+
+ Installing or uninstalling the :ref:`audit_log_plugin` would crash the server if the :variable:`audit_log_file` variable was pointing to an inaccessible path. Bug fixed :bug:`1435606`.
+
+ The :variable:`audit_log_file` would point to random memory area if the :ref:`audit_log_plugin` was not loaded into server, and then installed with ``INSTALL PLUGIN``, and :file:`my.cnf` contained :variable:`audit_log_file` setting. Bug fixed :bug:`1437505`.
+
+ Issuing a ``ROLLBACK TO SAVEPOINT`` within an `error handler <https://dev.mysql.com/doc/refman/5.6/en/declare-handler.html>`_ causes slave to crash with ``Assertion failed: table_found`` error. Bug fixed :bug:`1438990` (upstream :mysqlbug:`76727`).
```

Comment:
A specific trigger execution on the master server could cause a slave assertion error under row-based replication. The trigger would satisfy the following conditions: 1) it sets a savepoint; 2) it declares a condition handler which releases this savepoint; 3) the trigger execution passes through the condition handler. Bug fixed ...